.github/ISSUE_TEMPLATE/new-dbt-model.md

```diff
@@ -202,15 +202,15 @@ cat dbt/models/default/default.new_model.sql
          description: 10-digit PIN
        - name: year
          description: Year
-+    tests:
++    data_tests:
 +      - unique_combination_of_columns:
 +        name: new_model_unique_by_pin_and_year
 +        combination_of_columns:
 +          - pin
 +          - year
```

Comment:
Not totally sure what's causing the weird whitespace changes in this file, but nothing has changed other than renaming `tests` to `data_tests`.